spring-cloud-aws-samples/spring-cloud-aws-parameter-store-sample/src/main/resources/application.properties

```diff
@@ -0,0 +1,14 @@
+# importing single parameter
+spring.config.import=aws-parameterstore:/config/spring
+
+aws.paramstore.region=eu-central-1
```

Comment:
I think since we did not define a region in CDK setup, we also should not define it here.